src/App.jsx

```diff
@@ -1,33 +1,57 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getPreparedList(sourceList, query) {
+  let preparedList = [...sourceList];
+  const preparedQuery = query.trim().toLowerCase();
+  const isIncludes = (str, value) => str.toLowerCase().includes(value);
```

Comment:
```suggestion
  const checkIfIncludes = (str, value) => str.toLowerCase().includes(value);
```
![image](https://github.com/mate-academy/react_movies-list-filter-js/assets/96409563/d698d10f-598b-4c0a-b2f5-8aa33b4ce8a1)